src/screens/SignInScreen.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { SignInScreen } from './SignInScreen'
+
+import { act, cleanup, fireEvent, render } from '~utils/testUtils'
+
+afterEach(cleanup)
+
+describe('SignInScreen', () => {
+  it('should render SignIn screen with the key elements', () => {
+    const { getByPlaceholderText, getByRole, getAllByRole } = render(<SignInScreen />)
+    const emailInput = getByPlaceholderText('john@doe.com')
+    const passwordInput = getByPlaceholderText('password')
+    const confirmationCheckbox = getByRole('checkbox')
+    const buttons = getAllByRole('button')
+
+    expect(emailInput).toBeDefined()
+    expect(passwordInput).toBeDefined()
+    expect(confirmationCheckbox).toBeDefined()
+    expect(buttons.length).toBe(2)
+  })
+
+  it('should display errors on required fields', async () => {
+    const { getByTestId, findAllByText, getByPlaceholderText, update } = render(<SignInScreen />)
+    const emailInput = getByPlaceholderText('john@doe.com')
+    const passwordInput = getByPlaceholderText('password')
+    const signInButton = getByTestId('signInButton')
+
+    expect(signInButton).toBeDefined()
+    fireEvent.changeText(emailInput, '')
+    fireEvent.changeText(passwordInput, '')
+    fireEvent.press(signInButton)
+
+    await act(async () => {
+      update(<SignInScreen />)
+      const requiredInputs = await findAllByText('This field is required')
```

Comment:
Same as above, when i18n provider is different that `en` then the test would fail

Reply:
IMO: we shouldn't use text to find elements. Even in test examples